base/initdefs.jl

```diff
@@ -21,17 +21,18 @@ const ARGS = String[]
 """
     exit([code])
 
-Quit (or control-D at the prompt). The default exit code is zero, indicating that the
```

Comment:
Should `^D` still be mentioned here? It is mentioned in the key binding docs though.